crypto/blake256/internal/_asm/README.md

```diff
@@ -0,0 +1,37 @@
+_asm
+====
+
+[![Build Status](https://github.com/decred/dcrd/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/dcrd/actions)
+[![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
+[![Doc](https://img.shields.io/badge/doc-reference-blue.svg)](https://pkg.go.dev/github.com/decred/dcrd/crypto/blake256/internal/_asm)
+
+## Overview
+
+This provides a separate internal module for generating the assembly code that
+provides vector acceleration for the BLAKE-224 and BLAKE-256 block compression
+function.
+
+It uses Go to generate the assembly functions and related stubs to support SSE2,
+SSE4.1, and AVX for `amd64` via [avo](https://github.com/mmcloughlin/avo).
+
+The internal module ensures the specific version of avo used to generate the
+code is pinned and therefore entirely reproducible without adding an otherwise
+unnecessary dependency to the main public module.
+
+## Generating Code
+
+The assembly code can be generating by running the following command in this
+directory:
+
+```bash
+$ go generate
```

Comment:
Trying to regenerate the bindings with a `go.work` file on the root of the project dir (created by the contrib script) is giving me

```
main module (github.com/decred/dcrd/crypto/blake256) does not contain package github.com/decred/dcrd/crypto/blake256/internal/_asm
```

I suggest replacing this with `GOWORK=off go generate` to automatically deal with it.

Other

Reply:
Fair point.  I always do that since I run with a workspace too, but I figured not everyone would necessarily be running with one, so I didn't.  It works equally well with GOWORK=off in that case though, so updated.